client/js/screens/Card/Card.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, ImageBackground } from 'react-native';
 import PropTypes from 'prop-types';
 
 import styles from './styles';
 
 const Card = props => {
-  return <Text>This is Card.</Text>;
+  return (
+    <ImageBackground
+      // source={'../../assets/images/logo_bg@3x.png'}
+      style={{ width: '100%', height: '100%' }}
```

Comment:
styles should be in the styles.js file.

Reply:
This code is temporary. I just want to see if the bg is showing